erpnext/buying/doctype/purchase_order/purchase_order.py

```diff
@@ -415,6 +415,54 @@ def make_stock_entry(purchase_order, item_code):
 	stock_entry.get_items()
 	return stock_entry.as_dict()
```

Comment:
There is two methods make_stock_entry, make_rm_stock_entry for the same functionality, can you commonify it?

Reply:
I think `make_stock_entry` does not required anymore.

Reply:
If we don't want to maintain old way of stock entry creation for one finished good, we can just change if condition in JS to call the new modal in all cases and then remove the old method.

Reply:
Have checked the code and found that BOM No is used for reference only. Can we create the single stock entry against all the items without BOM No?

Reply:
In that case, what should we set in stock entry (finished good completed quantity, should it be sum of all finished goods item completed quantities? or should it be zero?)
stock_entry.fg_completed_qty = po_item.qty

Reply:
Can you remove the old method make_stock_entry, we can use method make_rm_stock_entry if one item is there.